Specs/DateTime/Hindi/SetExtractor.json

```diff
@@ -313,6 +349,7 @@
   },
   {
     "Input": "क्या मैं 09th मई को 2 रातों के लिए बुकिंग करवा सकता हूँ ",
+    "Comment": "(Can I do a booking for the 09th of May for 2 nights?). Hindi does not use plurals of weekdays such as 'Mondays', 'mornings', 'nights'",
```

Comment:
So there is no "equivalent" sentence that would generate a Set?
Also, based on the translation, it seems this spec should move to Model and output one date and one period?

Reply:
@LionbridgeCSII, is this an equivalent situation to the other spec question?

Reply:
similar, if I understand correctly, there is no difference in Hindi between 'night' and 'nights' and 'night' is also used in other contexts, for example as 'pm' indicator in time expressions. So, adding it to Set causes problems in other sub-entities. I am double checking if it is possible to support it in certain cases. 
The case is already included in the Model.